test/tests/classname.py

```diff
@@ -1,3 +1,5 @@
+# expected: fail
+
```

Comment:
I think we should try to add a note as to why the test is marked "expected: fail".  Also you can remove the notes from the test cases that are no longer failing :)